lib/hanami/utils/string.rb

```diff
@@ -602,7 +602,7 @@ def singularize
 
       # Returns the hash of the internal string
       #
-      # @return [Fixnum]
+      # @return [Hash]
```

Comment:
I'm gonna fix this. It returns an `Integer` (previous versions of Ruby had the `Fixnum` class).

```ruby
"Hanami".hash
  # => 3882157824217614589
```